test/e2e/common/configmap_volume.go

```diff
@@ -540,6 +540,19 @@ var _ = Describe("[sig-storage] ConfigMap", func() {
 		})
 
 	})
+
+	//Slow (~5 mins)
+	It("non-optional updates should not reflected in volume [Slow]", func() {
```

Comment:
@msau42 I will break down this test case to two. 

- pod fails because configmap object doesn't exist
- pod fails because the key in the configmap object doesn't exist